packages/circuit-ui/components/Toggle/Toggle.tsx

```diff
@@ -119,16 +114,19 @@ export const Toggle = forwardRef(
           id={switchId}
           ref={ref}
         />
-        <ToggleTextWrapper>
-          <ToggleLabel as="label" id={labelId} htmlFor={switchId}>
-            {label}
-          </ToggleLabel>
-          {explanation && (
-            <ToggleExplanation size="two" variant="subtle" id={explanationId}>
-              {explanation}
-            </ToggleExplanation>
+        <ToggleLabel as="label" id={labelId} htmlFor={switchId}>
+          {label}
+          {(description || explanation) && (
+            <FieldDescription aria-hidden="true">
```

Comment:
Just thinking: is `FieldDescription` the one with `aria-live`? Or this was added manually in other components?

Because it wouldn't have any effect in this case since we're passing `aria-hidden`.

Ideally the wrapper of the screen reader description (`p` below the label) should have `aria-live` to announce any updates (useful for validation). I think I did something like this for other form components

Reply:
No, this is a new component I added specifically for the description. You're thinking of the `FieldValidationHint`. The description isn't meant to change.